packages/bpk-component-rating/src/BpkRating.js

```diff
@@ -102,7 +102,12 @@ const BpkRating = (props: Props) => {
 
   return (
     // $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'.
-    <div className={classNames.join(' ')} aria-label={ariaLabel} {...rest}>
+    <div
+      className={classNames.join(' ')}
+      aria-label={ariaLabel}
+      role="img"
```

Comment:
This might not always be an img which would be incorrect if its text as would be a mis representation

This might be worth having it instead: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/article_role

or even: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/complementary_role